schutzbot/deploy.sh

```diff
@@ -33,6 +33,35 @@ sudo dnf install -y osbuild-composer composer-cli
 sudo mkdir -p /etc/osbuild-composer
 sudo cp -a schutzbot/osbuild-composer.toml /etc/osbuild-composer/
 
+sudo mkdir -p /etc/osbuild-worker
+
+# if GCP credentials are defined in the ENV, add them to the worker's configuration
+GOOGLE_APPLICATION_CREDENTIALS="${GOOGLE_APPLICATION_CREDENTIALS:-}"
+if [[ -n "$GOOGLE_APPLICATION_CREDENTIALS" ]]; then
+     # The credentials file must be copied to a different location. Jenkins places
+     # it into /tmp and as a result, the worker would not see it due to using PrivateTmp=true.
+     GCP_CREDS_WORKER_PATH="/etc/osbuild-worker/gcp-credentials.json"
+     sudo cp "$GOOGLE_APPLICATION_CREDENTIALS" "$GCP_CREDS_WORKER_PATH"
+     echo -e "\n[gcp]\ncredentials = \"$GCP_CREDS_WORKER_PATH\"\n" | sudo tee -a /etc/osbuild-worker/osbuild-worker.toml
+fi
+
+# if Azure credentials are defined in the env, create the credentials file
+AZURE_CLIENT_ID="${AZURE_CLIENT_ID:-}"
+AZURE_CLIENT_SECRET="${AZURE_CLIENT_SECRET:-}"
+if [[ -n "$AZURE_CLIENT_ID" && -n "$AZURE_CLIENT_SECRET" ]]; then
+     set +x
```

Comment:
Why the set +x here?

Reply:
to not to leak credentials in CI log

Reply:
It would dump the secrets in the logs otherwise but I have no idea if we take the same precautions in composer.

Reply:
Shouldn't Jenkins take care of this? It's set to mask these, isn't it?

Reply:
we do the same thing in composer. I basically copied the Azure part from composer tests 1:1

Reply:
Maybe @ondrejbudai can share his motivation as the original author of this piece of code in composer...